Cargo.lock

```diff
@@ -2912,9 +2912,10 @@ dependencies = [
 [[package]]
 name = "phylum_types"
 version = "0.1.0"
-source = "git+https://github.com/phylum-dev/phylum-types?branch=development#bb39c21c3d6f42f83219ff24bbe3ce3bdf4b25a1"
+source = "git+https://github.com/phylum-dev/phylum-types?branch=golang-support#0e6d0baa9e432e9f71710d9c047a1a74c6fa7cd1"
```

Comment:
As mentioned in the PR notes, this needs to be fixed before merging. I'm just marking it here so that it doesn't slip through the cracks.